Libraries/LibWeb/Fetch/Response.cpp

```diff
@@ -84,14 +84,27 @@ GC::Ref<Response> Response::create(JS::Realm& realm, GC::Ref<Infrastructure::Res
     return response_object;
 }
 
+// A status text is valid if it matches the [reason-phrase](https://httpwg.org/specs/rfc9112.html#status.line) token production.
+// reason-phrase  = 1*( HTAB / SP / VCHAR / obs-text )
+// VCHAR          = %x21-7E
+// obs-text       = %x80-FF
+static bool is_valid_status_text(StringView status_text)
```

Comment:
Please add the spec link before the function signature, and the spec comments should go inside the function body.